modules/preview-react/pill/lib/Pill.tsx

```diff
@@ -149,9 +149,10 @@ const StyledBasePill = styled(BaseButton.as('button'))<StyledType & PillProps>(
   boxStyleFn
 );
 
-const StyledNonInteractivePill = styled(StyledBasePill)({
+const StyledNonInteractivePill = styled(StyledBasePill)<StyledType & PillProps>({
```

Comment:
Do you need `PillProps`? I think that doubles the `PillProps` since they are already extracted from `StyledBasePill`